weldx/util.py

```diff
@@ -23,6 +23,36 @@
     import weldx.transformations as tf
 
 
+def deprecated(func: Callable):
+    """Mark a functions as deprecated.
+
+    This decorator emits a warning when the function is used.
+
+    Notes
+    -----
+    Original source: https://stackoverflow.com/a/30253848/6700329
+
+    Parameters
+    ----------
+    func :
+        The wrapped function
```

Comment:
In general I think we should always start by listing the `Parameters` and `Returns` sections first after the function/class description just to make it easy to spot in the documentation